test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -375,8 +375,8 @@ public function mergeableRangesProvider() : array
             ['>1,<2.10', '>1,<2.100', false, true],
             ['>1.0,<2', '>1,<2', true, true],
             ['>1,<2.0', '>1,<2', true, true],
-            ['>1.0.0,<2', '>1,<2', true, true],
-            ['>1,<2.0.0', '>1,<2', true, true],
```

Comment:
Same as above: shouldn't be a change, but rather an addition to the existing test